api/components/attributes/B2B_Dom/on_search/on_search.yaml

```diff
@@ -38,11 +66,44 @@ message:
         usage: "-"
         reference: if any
       items:
+        parent_item_id:
+          description: multiple items tags to single parent item id, in case of variants
+          required: optional
+          usage: "-"
+          reference: if any
+        category_id:
+          description: type of the category, should be mapped from categories
+          required: required
+          usage: "[C1]"
+          reference: if any
+        fulfillment_id:
+          description: indicates forward shipment item with quote;
+          required: required
+          usage: "[1]"
+          reference: if any
         descriptor:
           code: 
             description: type of shipment; enum -  "P2P" (point-to-point) and "P2H2P" (point-to-hub-to-point). P2H2P fulfillments require different packaging and AWB no. This is required for merchants to decide on the packaging required for shipment
             required: optional
             usage: "-"
             reference: if any
+          price:
+            description: price of the shipment
+            required: required
+            usage: "4000"
```

Comment:
This is not price of shipment. This is the cost of logistics to send goods via specified logistics.